recipes-browser/wpewebkit/wpewebkit/0001-Fix-build-with-musl.patch

```diff
@@ -0,0 +1,75 @@
+From c4d4d9f1aa74addefdad40294cf16d9e0b3dd6ec Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Fri, 6 Oct 2017 17:00:08 +0300
+Subject: [PATCH] Fix build with musl
+
+__GLIBC__ only covers glibc and uclibc C library on Linux but we need
+same condition for musl as well, which does not (intentionally) define
+libc specific builtin define
+
+Upstream-Status: Pending
```

Comment:
I don't like adding downstream patches that are not forwarded to upstream.
So, can you at least open an issue about this on bugs.webkit.org and update this patch with a link to it?

Reply:
A similar patch is already used by Alpine Linux for WebKitGTK: https://git.alpinelinux.org/aports/tree/community/webkit2gtk/musl-fixes.patch and for Void-Linux: https://github.com/void-linux/void-packages/blob/master/srcpkgs/webkit2gtk/patches/fix-musl-javascriptcore.patch

Reply:
I think the JSC part is required so I added that as separate patch and refreshed the pull. Take a look